### ISSUE TITLE
Add support for gevent 1.3 enhanced monitoring

### ISSUE THREAD

**Issue by millerdev** [DataDog/dd-trace-py#683]
The [async libraries page](http://pypi.datadoghq.com/trace/docs/#instrumented-libraries) says gevent >= 1.0, < 1.3 is supported. What does this mean for gevent 1.3? Is `ddtrace-run` expected work at all with it? What sort of symptoms should we expect if we are using the two together?

We use the following library versions:
```
ddtrace==0.14.1
django==1.11.16
gevent==1.3.7
gunicorn==19.9.0
```

Consider this a formal request to add support for gevent 1.3 if it is not supported yet.

**Comment by brettlangdon** [comment]
@millerdev we actually have gevent 1.3 support ready for our next release `0.16.0`.

The PR with the changes adding support is #663

**Comment by millerdev** [comment]
Excellent, and thank you for the prompt response, @brettlangdon. Are there any plans for adding support for gevent 1.3 [enhanced monitoring](http://www.gevent.org/monitoring.html)? I would love to see `EventLoopBlocked` events in datadog.

**Comment by brettlangdon** [comment]
hmm... interesting... This is my first time seeing this, but could be interesting to consider adding support for. I will surface this information to the team. However it isn't likely to make it into this next release :(

**Comment by aleffert** [comment]
when is that release coming out? this is currently blocking me and I'm on a tight schedule

**Comment by brettlangdon** [comment]
@aleffert my apologizes, we do not have a specific date locked down yet for the `0.16.0` release.

However, if you are interested in trying out the latest pre-release version you can install it via:

```
pip install git+https://github.com/datadog/dd-trace-py@0.16-dev
```

Or you can lock down to the most recent commit only (which is the merge for gevent 1.3 support)

```
pip install git+https://github.com/datadog/dd-trace-py@91d7052087d920b4331c210afaf5872e65f0ab93
```

If you do give this a try, please report back any issues you have so we can address them before our next release!

**Comment by majorgreys** [comment]
Closing since v0.16.0 was released in #704
